syntax = "proto3";

package sandbox;

message MergeMessage {
  string string_field_from_a = 1;
  string string_field_from_b = 2;
  int32 int32_field_from_a = 3;
  int32 int32_field_from_b = 4;
  bool bool_field_from_a = 5;
  bool bool_field_from_b = 6;

  MessageField message_field = 7;
  repeated MessageField repeated_message_field = 8;
  repeated int32 repeated_int32_field = 9;

  message MessageField {
    string from_a = 1;
    string from_b = 2;
  }
}
